src/fl.go

```diff
@@ -62,12 +62,12 @@ func noTui(Flags helpers.FlagStruct, Config io.Config) {
 
 	// if not skipping prompt, ask user if they would like to execute
 	userExecute := false
-	if Flags.PromptExec && !Config.Autoexec {
+	if Flags.PromptExec {
 		userExecute = exec.PromptExec()
 	}
 
 	// perform the command if autoexecute enabled or user prompted to exec
-	if Config.Autoexec || userExecute {
+	if (Config.Autoexec && !Flags.PromptExec) || userExecute {
```

Comment:
why do you need this predicate `&& !Flags.PromptExec`?

Reply:
i made changed the logic such that if -p is passed, the prompt to execute will always show. 

otherwise, the prompt is skipped and follows autoexecution or no execution based on `Config.autoexecute`